bundles/org.openhab.binding.buienradar/README.md

```diff
@@ -50,3 +55,20 @@ Number RAIN_115MIN "Rain 115 min." (Rain) {channel="buienradar:rain_forecast:hom
 Number RAIN_120MIN "Rain 120 min." (Rain) {channel="buienradar:rain_forecast:home:forecast_120" }
 ```
 
+## Example data visualisation
+
+In dit voorbeeld gebruiken we de 'Discrete' plugin van Grafana om een visualisatie te maken van de voorspellingen. Er zijn mappings gemaakt naar neerslagintensiteit (licht, middelmatig, zwaar) en die categorieen krijgen de desbetreffende kleuren.
+
+![Z-Way Binding](doc/img/grafana-dashboard.png)
+
+De mappings zijn als volgt:
```

Comment:
see above

Reply:
Would you believe I copy-pasted the wrong side of the [deepl translator](deepl.com)? Fixed anyways.